new-pipeline/evo.html

```diff
@@ -54,6 +54,13 @@ <h2 style="font-size: 18px; margin-bottom: 0">
                     as
                     <select id="filter-process-type" class="multiselect" title="Process type" data-all-selected="any process type"></select>
                 </h2>
+                <hr></hr>
+                <div class="caption-div">
+                <figcaption id="evo-caption" class="overflow ellipsis"></figcaption>
+                <figcaption id="evo-caption-link"></figcaption>
+                </div> 
+                <h2 style="font-size: 18px; margin-bottom: 0">Showing graphs for <select id="selected-key" class="multiselect" title="Selected key"></select></h2>
```

Comment:
This is duplicated, you can see [here](https://bit.ly/2FBu2ac) how it shows up two times.

Reply:
Yeah, I see.